www/header.php

```diff
@@ -39,24 +41,47 @@
 
     <!-- RESOURCES -->
     <!-- Common CSS -->
+	<!-- build:css css/styles.min.css -->
```

Comment:
the build and enbuild tag are use to extract which resources are used and to which bundle it should be added.
A new header.php is generated which contains the bundles,